node-notes-cli/delete-data.js

```diff
@@ -0,0 +1,15 @@
+const fs = require('fs');
+const object = require('./data.json');
+if (process.argv[2] === 'delete') {
+  for (var key in object.notes) {
+    if (key === process.argv[3]) {
+      object.nextId--;
+      delete object.notes[key];
+    }
+  }
```

Comment:
We don't want to reset the `nextId` at all. Once a note is deleted, it's gone forever, including its id.

Also, we don't need a loop, since we already have the target id

```js
delete object.notes[process.argv[3]];
```